src/index.js

```diff
@@ -1,17 +1,41 @@
 import express from 'express';
 import cors from 'cors';
-
 // Create global app object
 const app = express();
+const bodyParser = require('body-parser');
 
 app.use(cors());
 
 // Normal express config defaults
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
 
```

Comment:
@funmi5 i think we can either stick with bodyparser or express in this block of code, I think they handle are meant for similar purposes... or what do you think??

Reply:
I tottally agree @oyewoas. express actually uses bodyParser under the hood, so it's like installing bodyParser twice

Reply:
OK @oyewoas  @darasimiolaifa